src/main/java/com/github/jcustenborder/kafka/connect/redis/RedisSinkConnectorConfig.java

```diff
@@ -32,14 +32,24 @@
   public final static String CHARSET_CONF = "redis.charset";
   public final static String CHARSET_DOC = "The character set to use for String key and values.";
 
+  public final static String CONNECTION_RETRY_CONF = "redis.connection.retry";
+  public final static String CONNECTION_RETRY_DOC = "The number of attempt when connecting to redis";
+
+  public final static String CONNECTION_RETRY_PAUSE_CONF = "redis.connection.retryPause";
```

Comment:
I tend to not use camel case by convention for configuration settings. Can you change this to `redis.connection.retry.delay.ms`?